### DISCUSSION TITLE
<Add Training and Validation data frames and ask copilot to implemnt and validate changes>

### DISCUSSION THREAD

**Discussion by ghost** [community/community#45804] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hi, anyone knows if I can upload a training data frame to copilot, ask copilot to implement a binary classification machine learning model, up load a validation data frame for copilot  to implement the model and if it overfits ask copilot to tune / change / torture the ml model to get better results?

**Comment by mruigomezc** [comment]
Hi,
I would like to know too if it is possible to upload training repositories in order to use the AI with my private repositories, not with the public ones. I joined to this thread. Thanks

**Comment by github-actions** [comment]
🕒 **Stale Discussion Alert** 🕒

This Discussion has been labeled as stale by an automated system for having no activity in the last 60 days. Please consider one the following actions:

1️⃣ Close as Out of Date: If the topic is no longer relevant, close the Discussion as `out of date` at the bottom of the page.

2️⃣ Provide More Information: Share additional details or context — or let the community know if you've found a solution on your own.

3️⃣ Mark a Reply as Answer: If your question has been answered by a reply, mark the most helpful reply as the solution.

Note: This stale notification will only apply to Discussions with the `Question` label. To learn more, see our recent stalebot announcement.

Thank you for helping bring this Discussion to a resolution! 💬
